src/main/java/com/fabriccommunity/spookytime/mixin/AbstractSkeletonEntityMixin.java

```diff
@@ -18,4 +25,13 @@
     private ItemStack getStack(AbstractSkeletonEntity entity, EquipmentSlot equipmentSlot) {
         return MixinHelpers.getEquippedOrPumpkin(entity, equipmentSlot);
     }
+    
+    @Inject(method = "initEquipment", at = @At(value = "TAIL"))
+    protected void initEquipment(LocalDifficulty ld, CallbackInfo cb) {
+        Random random = new Random();
+        if (random.nextInt(3) == 0) {
```

Comment:
The random should probably be obtained from the entity itself, and the limit of nextInt should be like 8